README.md

```diff
@@ -1,5 +1,9 @@
 # uwanjani
 Playing around with git 
 
-##Task 1
+## Task 1
 Songesha branch ui mbele ya master (kidogo tu)
+
+## Task 2
+Make a new branch from ui. Call it ui-homepage
+I made this line from ui-homepage
```

Comment:
Maybe you should add that you intend to merge ui-homepage to ui

Reply:
Thank you @wandesky-tests but I think I will keep it that way. One step at a time. 😞